tests/unit/quoted.test.scala

```diff
@@ -1,27 +1,43 @@
 // SYNTAX TEST "source.scala"
 
+   'x
+// ^ keyword.control.flow.staging.scala constant.other.symbol.scala
+//  ^ constant.other.symbol.scala
    '{ 2 }
-// ^^ punctuation.section.block.begin.scala
+// ^ keyword.control.flow.staging.scala
```

Comment:
Actually, should `{` have some kind of scope like in imports/exports? (for example we have `meta.export.scala meta.export.selector.scala meta.bracket.scala`) Or it doesn't really matter since braces are anyway interpreted correctly?

Reply:
`{` have `punctuation.section.block.begin.scala` and `[` have `meta.bracket.scala`.

I updated the tests

Reply:
Thanks!